### PR TITLE
Add TSC compliance section

### PR DESCRIPTION
**Pull request by javimed**
## Description
This PR adds the new TSC compliance section.

## Checks
- [ ] Compiles without warnings.
- [ ] Uses present tense, active voice, and semi-formal registry.
- [ ] Uses short, simple sentences.
- [X] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.
- [X] Uses three spaces indentation.
- [X] Adds or updates meta descriptions accordingly.
- [X] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/images/compliance/tsc/common-criteria/events-alerts-review.gif
Comment (s-ocando) [inline source/images/compliance/tsc/common-criteria/events-alerts-review.gif:1]: Improve GIF to shorten the loading time of the Events tab.

File: source/compliance/tsc/index.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+This document outlines use cases that show how Wazuh helps users comply with the TSC common criteria, and the additional criteria for availability, confidentiality, and processing integrity. We have also created the `Using Wazuh for TSC 2017 requirements guide <https://documentation.wazuh.com/resources/using-wazuh-for-TSC-2017-requirements-guide.pdf>`__, which complements this document. Please refer to the guide for more details on how Wazuh helps meet TSC requirements.
+
+The following sections outline some of the technical requirements that Wazuh supports:
+
+.. toctree::
+   :caption: The COSO Principle and Common Criteria
+   :maxdepth: 1
+
+   common-criteria/cc2.1
```
Comment (s-ocando) [inline source/compliance/tsc/index.rst:49]: Define specific texts for the side menu as the section titles are too long. 

image

**Review by s-ocando — COMMENTED** [review]
Hi @javimed, please review my suggestions below.

File: source/images/compliance/tsc/common-criteria/security-events-dashboard.png
[the remark this answers is not in the document]
Comment (javimed) [reply source/images/compliance/tsc/common-criteria/security-events-dashboard.png:1]: Image replaced

File: source/images/compliance/tsc/common-criteria/events-alerts-review.gif
Comment (javimed) [reply source/images/compliance/tsc/common-criteria/events-alerts-review.gif:1]: GIF replaced

File: source/compliance/tsc/common-criteria/cc2.1.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+#. Click on **Security events** from the Wazuh dashboard:
+
+.. thumbnail:: /images/compliance/tsc/common-criteria/wazuh-dashboard.png
+   :title: Wazuh dashboard
+   :alt: Wazuh dashboard
+   :align: center
+   :width: 80%
+
+The image below shows the Wazuh Security events dashboard.
```
Comment (s-ocando) [inline source/compliance/tsc/common-criteria/cc2.1.rst:37]:
```suggestion
Check the security events on the dashboard and click on any alert to view more details. 
```

**Review by s-ocando — APPROVED** [review]

**Pull request merged**